server/server.js

```diff
@@ -0,0 +1,98 @@
+var Launchpad = require('api.js').Launchpad;
+var express = require('express');
+var jira = require('jira');
+var crypto = require('crypto');
+var CancellablePromise = require('metal-promise').CancellablePromise;
+
+var CRYPTO_ALGORITHM = 'AES-256-CBC';
+
+var CRYPTO_PASSWORD = 'nodegh.io';
+
+var app = express();
+
+var jiraConfig = {
+	protocol: 'https',
+	host: 'issues.liferay.com',
+	user: 'bruno.basto',
+	password: 'cc0aabf9f2ed6ade85796e8c7633a439',
+	api_version: '2'
+}
+
+var decryptText_ = function(text) {
+	var decipher,
+		decrypted;
+
+	decipher = crypto.createDecipher(CRYPTO_ALGORITHM, CRYPTO_PASSWORD);
+
+	decrypted = decipher.update(text, 'hex', 'utf8');
+
+	decrypted += decipher.final('utf8');
+
+	return decrypted;
+};
+
+var JiraApi = new jira.JiraApi(
+	jiraConfig.protocol,
+	jiraConfig.host,
+	jiraConfig.port,
+	jiraConfig.user,
+	decryptText_(jiraConfig.password),
+	jiraConfig.api_version
+);
+
+app.use(express.static('public'));
+app.use('/build', express.static('build'));
+app.use('/dependencies', express.static('node_modules'));
```

Comment:
@brunobasto that's awesome! I didn't know this feature. Living and learning. 👍 :)